packages/components/src/__future__/Select/Select.tsx

```diff
@@ -195,6 +195,7 @@ export const Select = <Option extends SelectOption = SelectOption>({
             id={popoverId}
             portalContainer={portalContainer}
             refs={refs}
+            data-testid="kz-select-popover"
```

Comment:
Any chance we can avoid baking in a test id? 😅 
Also, I thought you mentioned it should have `role="listbox"` or am I imagining things? 😬

Reply:
Not unless we add a prop to Select which isn't better.

Listbox is included but it's many layers away.

We could also just delete the tests because they're not really testing anything with any user impact.

Reply:
I've removed it now by removing the test that checks the default behaviour, and adjusting the last one to just check that it still renders somewhere when passed a bad id.